index.js

```diff
@@ -16,29 +52,29 @@ const createRequest = (input, callback) => {
             "Authorization": "Bearer " + process.env.API_KEY,
             "Content-Type": "application/json"
         },
-        json: true
+        timeout: timeout,
+        json: true,
+        resolveWithFullResponse: true
     }
-    request(options, (error, response, body) => {
-        if (error || response.statusCode >= 400) {
-            const statusCode = response ? response.statusCode : 500
-            console.log(error)
-            const errorText = error || body
-            callback(statusCode, {
-                jobRunID: input.id,
-                status: "errored",
-                error: errorText,
-                data: {}
-            });
-        } else {
+    requestRetry(options, retries)
+        .then(response => {
+            const result = response.body["overallVWAP"]
```

Comment:
This basically disables getting any other information from the call. Not as flexible, but works for me.

Reply:
This does not actually prevent other information from being included in the response, it just pre-selects the value we usually want. This makes it so that we don't need the copy adapter – or we can run the copy adapter without parameters.

You will still be able to run the copy adapter with any parameters that will select any of the other keys in the response.

Reply:
Ahh, now I see, there's `result` and `data` in the reply. Gotcha.